src/api/server.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export async function fetchTodoList() {
+    const response = await fetch('http://127.0.0.1:8000/todo-list/')
```

Comment:
You'll need this URI to be configurable - the server won't always be running locally.

Reply:
Will update.